purchases/src/main/kotlin/com/revenuecat/purchases/paywalls/components/common/ExitOffersConfiguration.kt

```diff
@@ -0,0 +1,21 @@
+package com.revenuecat.purchases.paywalls.components.common
+
+import com.revenuecat.purchases.InternalRevenueCatAPI
+import dev.drewhamilton.poko.Poko
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@InternalRevenueCatAPI
+@Poko
+@Serializable
+class ExitOffersConfiguration(
```

Comment:
I am using ExitOffers & ExitOffer which I am not sure I like. Not a fan of `Configuration` either hehe 💅